content/W2015/W15-event-profsdinner.md

```diff
@@ -0,0 +1,23 @@
+Title: Dinner With Profs
+Date: 2015-03-26 5:30PM
+Category: Events
+Tags: profs, dinner
+Slug: dinner-with-profs
+Author: Anna Lorimer
+Summary: WiCS and WiM are hosting a Dinner with Profs event! Come get to know other successful women in your field. 
+
+WiCS and WiM would like to invite all undergraduate ciswomen and transwomen, and non-binary folk to our Dinner with Profs event!
```

Comment:
Now that we have the FAQ, I think we can just link to that re: "women" rather than write this out explicitly every time we advertise an event.

Also, I prefer "cis" and "trans" as adjectives; "transgendered" or "transwoman" as a noun is usually not preferred by my trans friends in my experience.